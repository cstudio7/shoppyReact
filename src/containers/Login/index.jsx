import React, { Component } from 'react';
import AuthForm from '../../components/AuthForm';

export default class Login extends Component {
  render() {
    return (
      <div>
        <AuthForm />
      </div>
    );
  }
}
