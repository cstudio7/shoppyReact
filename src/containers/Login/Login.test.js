import React from 'react';
import { shallow } from 'enzyme';
import Login from './index';

describe('Test the AdminNav Component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Login />);
    expect(wrapper).toMatchSnapshot();
  });
});
