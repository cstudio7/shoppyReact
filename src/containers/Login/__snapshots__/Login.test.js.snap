// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Login Component should match snapshot 1`] = `
<div>
  <AuthForm />
</div>
`;
